realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -404,6 +406,192 @@ public boolean isValid() {
         return this;
     }
 
+    // In
```

Comment:
Uh.. never mind, it was my fault. I thought it is a unfinished javadoc.
Sorry :grin: !!! Just leave it here.